kong/plugins/http-log/handler.lua

```diff
@@ -189,12 +190,27 @@ function HttpLogHandler:log(conf)
 
   local queue_conf = Queue.get_plugin_params("http-log", conf, make_queue_name(conf))
   kong.log.debug("Queue name automatically configured based on configuration parameters to: ", queue_conf.name)
+  local entry = cjson.encode(kong.log.serialize())
```

Comment:
Pay attention to @chronolaw PR: https://github.com/Kong/kong/pull/13421

We now use the `clustering_utils.json_decode` or `clustering_utils.json_encode`.